src/main/java/com/cburch/logisim/util/LineBuffer.java

```diff
@@ -107,7 +107,7 @@ public LineBuffer withHdlPairs() {
         .pair("bracketClose", HDL.BracketClose())
         .pair("<", HDL.BracketOpen())
         .pair(">", HDL.BracketClose())
-//        .pair("else", HDL.())         // TODO
+        // .pair("else", HDL.())         // TODO
```

Comment:
It's OK, while this is draft, but please do not merge commented out code.

Reply:
As mentioned in HDL.java I'm currently searching for a good name ^^'
Any ideas?